src/engraving/infrastructure/draw/font.h

```diff
@@ -35,10 +35,12 @@ class Font
     Font(const QString& family = QString());
 
     enum class Style {
-        Undefined   = 0,
+        //Undefined   = -1;
+        Normal      = 0,
```

Comment:
Bringing it into sync with src/engraving/libmscore/types.h and src/inspector/types/texttypes.h